### PR TITLE
[COTCO-489][AC] Remove Vessel from the Copy to Email

### PR DESCRIPTION
**Pull request by andreicostin**

### CODE REVIEW

File: app/uk/gov/hmrc/ctc/views/transits.scala.html [hunk 1]
```diff
@@ -45,9 +45,6 @@
     @Messages("transit.label.from") @transit.departurePort,<br/>
     @Messages("transit.label.scanDateTime"): @DateTimeFormatter.ofPattern("d MMM yyyy HH:mm").withLocale(Locale.UK).withZone(ZoneId.systemDefault()).format(transit.captureDateTime),<br/>
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/views/transits.scala.html]: This line should be changed... you should use the `formatDate` function

Comment (andreicostin) [reply app/uk/gov/hmrc/ctc/views/transits.scala.html]: is the formDate available in this branch too?

Comment (udinobi) [reply app/uk/gov/hmrc/ctc/views/transits.scala.html]: Yes. The function is already in the template, at line 33.
You can search `formatDate` within the template to figure out how to use it.

Comment (andreicostin) [reply app/uk/gov/hmrc/ctc/views/transits.scala.html]: done!

**Pull request merged**